src/tools/csi-kata-directvolume/deploy/kata-directvolume/csi-directvol-plugin.yaml

```diff
@@ -87,7 +87,8 @@ spec:
 
         - name: kata-directvolume
           # build and push it into registry
-          image: localhost/kata-directvolume:v1.0.18
```

Comment:
Additionally, the directory name, `pod-with-directvol3` is not good either. Please rename it to something more meaningful.